File: src/org/cappuccino_project/ide/intellij/plugin/psi/PsiMissingSemiColonErrorElement.java
package org.cappuccino_project.ide.intellij.plugin.psi;

import com.intellij.psi.impl.source.tree.PsiErrorElementImpl;

public class PsiMissingSemiColonErrorElement extends PsiErrorElementImpl {
    private static final String ERROR_DESCRIPTION = "Missing semi-colon";
    public PsiMissingSemiColonErrorElement() {
        super(ERROR_DESCRIPTION);
    }
}

File: src/org/cappuccino_project/ide/intellij/plugin/psi/PsiMissingSemiColonErrorElement.kt
package org.cappuccino_project.ide.intellij.plugin.psi

import com.intellij.psi.impl.source.tree.PsiErrorElementImpl

class PsiMissingSemiColonErrorElement : PsiErrorElementImpl(ERROR_DESCRIPTION) {
    companion object {
        private val ERROR_DESCRIPTION = "Missing semi-colon"
    }
}
